i18n/index.js

```diff
@@ -1,32 +1,32 @@
-import path from 'path';
-import LanguageDetector from 'i18next-browser-languagedetector';
-import NextI18Next from 'next-i18next';
-import { initReactI18next } from 'react-i18next';
-import getConfig from '../tenant.config';
+// import path from 'path';
+// import LanguageDetector from 'i18next-browser-languagedetector';
```

Comment:
You should delete unused lines and remove unused packages from package.json as well:
```
    "i18next": "^19.9.2",
    "i18next-browser-languagedetector": "^6.1.2",
```